docs/en/samples/automatic_clean_up_data_operation.md

```diff
@@ -0,0 +1,112 @@
+# Demo - Automatic Cleanup Data Operation
+
+## Background
+
+Fluid's universal data operations describe operations such as data prefetch, data migration, elastic scaling, cache cleaning, metadata backup, and recovery.
+Similar to the Kubernetes Job's automatic cleaning mechanism, we also provides automatic cleanup data operation, utilizing the Time-to-Live (TTL) mechanism to limit the lifecycle of data operations that have finished execution. This document will briefly demonstrate the utilization of these features.
+
+
+## Prerequisites
+Before everything we are going to do, please refer to [Installation Guide](../userguide/install.md) to install Fluid on your Kubernetes Cluster, and make sure all the components used by Fluid are ready like this:
```

Comment:
In the "Prerequisites" section, I recommend using "Before we start" instead of "Before everything we are going to do" to make the sentence more succinct. Also, consider adding "Make sure that all the components required by Fluid are set up correctly" to instruct the reader clearly.